tests/storage/README.md

```diff
@@ -9,18 +9,29 @@ or fail (depending on the test).
 
 To setup storage for these tests, run:
 
-    python tests/storage/userstorage.py setup
+    make storage
 
 This can be run once when setting up development environment, and must
 be run again after rebooting the host.
 
+The storage is configured in the file `storage.py` in the root of the
+project. Tests that use user storage load this configuration file and
+access the storage via the BACKENDS dict.
+
+Some storage may not be available on all environments. For example on
+CentOS 7 userstorage cannot create storage with 4k sector size, and in
+oVirt CI, this usually fails. Tests should check if storage exists and
+mark test as xfail or skip when the storage is not available.
```

Comment:
This section is outdated (limit of old ovirt ci, based on centos 7) and can be removed.